source/configuration_manual/namespace/index.rst

```diff
@@ -87,36 +104,52 @@ When ``yes`` namespace will be hidden from IMAP ``NAMESPACE`` command.
 -------------------------------
 
  - Default: ``no``
- - Values: :ref:`boolean`
+ - Values:  :ref:`boolean`
+
+If :ref:`setting-namespace_location` fails to load, by default the entire
+session will fail to start.
 
-If namespace :ref:`setting-namespace_location` fails to load, this namespace will not be included.
-Otherwise the whole sessio will fail to start.
+If this is set, this namespace will be ignored instead.
 
 
 .. _setting-namespace_inbox:
 
 ``namespace/inbox``
 -------------------
 
- - Default: ``no``
- - Values: :ref:`boolean`
+- Default: ``no``
+- Values:  :ref:`boolean`
 
-When ``yes`` this namespace will be considered the one holding the ``INBOX`` folder.
-There can be only one namespace like this.
+If set, this namespace will be considered the one holding the INBOX folder.
+
+There can be only one namespace defined like this.
 
 
 .. _setting-namespace_list:
 
 ``namespace/list``
 ------------------
 
- - Default: `yes`
- - Values: :ref:`boolean`
+- Default: ``yes``
+- Values:  ``yes``, ``no``, or ``children``
+
+Include this namespace in LIST output when listing its parent's folders.
+
+Options:
+
+============= ==============================================================
+Value         Description
+============= ==============================================================
+``children``  Namespace prefix list listed only if it has child mailboxes.
+``no``        Namespace and mailboxes not listed.
+``yes``       Namespace and mailboxes listed when the namespace prefix isn't
+              explicitly specified as a search parameter.
+============= ==============================================================
 
-Whether to include this namespace in LIST output, when listing it's parent's folders.
-It is still possible to list the namespace's folders by explicitly asking for them.
-For example with namespace prefix ``lazy-expunge/`` using ``LIST "" *`` won't list it,
-but using ``LIST "" lazy-expunge/*`` lists all folders under it.
+It is still possible to list the namespace's folders by explicitly asking for
+them. For example, if this setting is ``no``, using ``LIST "" *`` with
+namespace prefix won't list it, but using ``LIST "" lazy-expunge/*`` lists all
```

Comment:
with namespace prefix \`\`lazy-expunge/\`\` - you dropped the lazy-expunge there.